tests/test_route.py

```diff
@@ -6,80 +6,66 @@
 
 from swsscommon import swsscommon
 
-
-class TestRoute(object):
+class TestRouteBase(object):
     def setup_db(self, dvs):
-        self.pdb = swsscommon.DBConnector(0, dvs.redis_sock, 0)
-        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
-        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.pdb = dvs.get_app_db()
+        self.adb = dvs.get_asic_db()
+        self.cdb = dvs.get_config_db()
 
     def set_admin_status(self, interface, status):
-        tbl = swsscommon.Table(self.cdb, "PORT")
-        fvs = swsscommon.FieldValuePairs([("admin_status", status)])
-        tbl.set(interface, fvs)
+        self.cdb.update_entry("PORT", interface, {"admin_status": status})
         time.sleep(1)
 
     def create_vrf(self, vrf_name):
-        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_VIRTUAL_ROUTER")
-        initial_entries = set(tbl.getKeys())
+        initial_entries = set(self.adb.get_keys("ASIC_STATE:SAI_OBJECT_TYPE_VIRTUAL_ROUTER"))
 
-        tbl = swsscommon.Table(self.cdb, "VRF")
-        fvs = swsscommon.FieldValuePairs([('empty', 'empty')])
-        tbl.set(vrf_name, fvs)
+        self.cdb.create_entry("VRF", vrf_name, {'empty': 'empty'})
```

Comment:
```suggestion
        self.cdb.create_entry("VRF", vrf_name, {"empty": "empty"})
```

Would recommend sticking to double-quotes throughout for consistency.

Reply:
Fixed.